BackofficeBundle/Resources/public/less/editTable.less

```diff
@@ -1,7 +1,40 @@
-.back-to-list {
-    margin-bottom: 20px;
+#widget-grid {
+
+    .submit_form, .submit_form:active, .back-to-list, .back-to-list:active, .bh-button-add, .bh-button-add:active {
+        width: 119px;
+        position: fixed;
+        top: 54px !important;
+        z-index: 901;
+    }
+    .back-to-list {
+        right: 15%;
+
+    }
+    .back-to-list:active, .submit_form:active, .bh-button-add:active {
+        left: 75.7% !important;
+    }
+    .submit_form:active, .bh-button-add:active {
+        left: 85.7% !important;
+    }
+    .submit_form, .bh-button-add{
+        right: 5% !important;
+    }
+    .modal-upload {
+        .back-to-list {
+            top: 17px !important;
+            right: 5% !important;
```

Comment:
Again a mix of pixels and % generally lead to something broken on other resolutions than the one you used to developped.